Examples/swift.mk

```diff
@@ -13,12 +13,23 @@ endif
 
 include $(SDK)/C_API/buildsupport/common.mk
 
-ifeq ($(TOOLCHAINS),)
-$(error Swift nightly toolchain not found; set ENV value TOOLCHAINS)
+# Determine the Swift toolchain by order of preference:
+#
+# 1. the pressence of a TOOLCHAINS environment value
+# 2. a Swift toolchain installed for the current user
+# 3. a Swift toolchain installed for all users
+TOOLCHAIN_PATH	= Library/Developer/Toolchains/swift-latest.xctoolchain
+ifneq ($(TOOLCHAINS),)
+else ifneq ($(wildcard $(HOME)/$(TOOLCHAIN_PATH)),)
+TOOLCHAINS = $(shell plutil -extract CFBundleIdentifier raw -o - $(HOME)/$(TOOLCHAIN_PATH)/Info.plist)
+else ifneq ($(wildcard /$(TOOLCHAIN_PATH)),)
+TOOLCHAINS = $(shell plutil -extract CFBundleIdentifier raw -o - /$(TOOLCHAIN_PATH)/Info.plist)
+else
+$(error Swift toolchain not found; set ENV value TOOLCHAINS (e.g. TOOLCHAINS=org.swift.59202403121a make))
```

Comment:
Can we add a `$(info ...)` which prints the path to the toolchain being used? I think this could help debug issues where swift-latest isn't what the user may expect. 

Otherwise this is great improvement, thank you!

Reply:
Good point, I'll add that in 👍